db.go

```diff
@@ -69,7 +69,10 @@ func NewDB(dsn string) (*DB, error) {
 	if err != nil {
 		return nil, err
 	}
-	db.SetMaxOpenConns(100)
+	db.SetMaxOpenConns(200)
+	db.SetMaxIdleConns(200)  // default is 2: https://golang.org/src/database/sql/sql.go#L652
```

Comment:
Yeah, https://github.com/mozilla-services/tigerblood/pull/55/commits/41db3806cb86363bde5c009f43f187f676f8ab10 makes it configurable. This is just bumping up the default.